src/ExpressionEvaluator/Core/Source/ExpressionCompiler/PseudoVariableUtilities.cs

```diff
@@ -54,5 +54,14 @@ internal static DkmClrCompilationResultFlags GetLocalResultFlags(this Alias alia
                     return DkmClrCompilationResultFlags.None;
             }
         }
+
+        internal static bool IsReturnValue0(this Alias alias)
+        {
+            int index;
+            return
+                alias.Kind == DkmClrAliasKind.ReturnValue &&
+                TryParseReturnValueIndex(alias.FullName, out index) &&
+                index == 0;
+        }
```

Comment:
Does this return `true` for `$ReturnValue` and `$ReturnValue0`? Is that intended?